### ISSUE TITLE
Improve heuristic optimisation

### ISSUE THREAD

**Issue by mitchelloharawild** [mitchelloharawild/fasster#34]
Potential directions include loess and sampling techniques.
Automatic model specification should also be considered with this.

**Comment by mitchelloharawild** [comment]
Another idea for this is to filter from the start of the series until some criteria is met, and then iterate this procedure using the new initial states until some other condition is satisfied. This should substantially speed up estimation for long time series.

**Comment by konradsemsch** [comment]
Do I understand correctly that ARMA terms are not selected automatically? What would be then the recommended way to go about that in an automated manner for a large number of time-series at the same time?

**Comment by mitchelloharawild** [comment]
Correct. Support for ARMA terms is currently limited to their functionality in `dlm::dlmModARMA()`. I would recommend against including ARMA terms in fasster models at this time.

Currently fasster does not provide tools for automatically identifying required model terms (#50). If the large number of time series exhibit similar structures, you can apply the same model definition to each series.
